actions/actions.py

```diff
@@ -107,18 +196,31 @@ def restore() -> None:
                                     "{}.key".format(client.common_name))
             with open(cert_file, 'r') as file:
                 cert = file.read()
-                print('writing cert to '
-                      '{}:\n{}'.format(client.common_name, cert))
             with open(key_file, 'r') as file:
                 key = file.read()
             client.set_cert(cert, key)
 
         except FileNotFoundError:
-            print("Cert not found for {}".format(client.common_name))
-
-
-def list_backups() -> None:
-    file_list: List[str] = []
+            if client.cert_type == 'client':
+                cert, key = create_client_certificate(client.common_name)
+                client.set_cert(cert, key)
+            elif client.cert_type == 'server':
+                cert, key = create_server_certificate(client.common_name,
+                                                      client.sans,
+                                                      client.common_name)
+                client.set_cert(cert, key)
```

Comment:
Suggestion to use hookenv.log every time data is shared to help troubleshooting.